kettle/make_db.py

```diff
@@ -39,6 +39,12 @@ def pad_numbers(string):
     return re.sub(r'\d+', lambda m: m.group(0).rjust(16, '0'), string)
 
 WORKER_CLIENT = None  # used for multiprocessing
+GARBAGE_JOBS = (
+    'pr-e2e-gce',
```

Comment:
what even is this?

Reply:
I think we can remove this https://github.com/kubernetes/test-infra/search?q=pr-e2e-gce&unscoped_q=pr-e2e-gce

Reply:
Both of the Garbage Garbage jobs removed.